From 3481d5fc5a1b5558c55b4bf81cba67e1401133c6 Mon Sep 17 00:00:00 2001
From: Jeffrey Mouroux <jeff.mouroux@xilinx.com>
Date: Mon, 28 Aug 2017 14:56:50 -0700
Subject: [PATCH 359/532] Documentation: bindings: devicetree: dma: Reformatted
 spaces to tabs

commit 486d5f6a0446808e6bfd0da0ec483e78c1e7a9a2 from
https://github.com/Xilinx/linux-xlnx.git

The original device tree used 8 leading spaces in the example
dts bindings.  This change reformats these to tabs for proper
indentation.

Signed-off-by: Jeffrey Mouroux <jmouroux@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 .../bindings/dma/xilinx/xilinx_frmbuf.txt          | 24 +++++++++++-----------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
index 8517ba0..c3465fc 100644
--- a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
+++ b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
@@ -25,21 +25,21 @@ Examples
 FB Read Example:
 ++++++++
 v_frmbuf_rd_0: v_frmbuf_rd@80000000 {
-        #dma-cells = <1>;
-        compatible = "xlnx,axi-frmbuf-rd-1.00.a";
-        interrupt-parent = <&gic>;
-        interrupts = <0 92 4>;
-        reset-gpios = <&gpio 80 1>;
-        reg = <0x0 0x80000000 0x0 0x10000>;
+	#dma-cells = <1>;
+	compatible = "xlnx,axi-frmbuf-rd";
+	interrupt-parent = <&gic>;
+	interrupts = <0 92 4>;
+	reset-gpios = <&gpio 80 1>;
+	reg = <0x0 0x80000000 0x0 0x10000>;
 };
 
 FB Write Example:
 ++++++++
 v_frmbuf_wr_0: v_frmbuf_wr@80000000 {
-        #dma-cells = <1>;
-        compatible = "xlnx,axi-frmbuf-wr-1.00.a";
-        interrupt-parent = <&gic>;
-        interrupts = <0 92 4>;
-        reset-gpios = <&gpio 80 1>;
-        reg = <0x0 0x80000000 0x0 0x10000>;
+	#dma-cells = <1>;
+	compatible = "xlnx,axi-frmbuf-wr";
+	interrupt-parent = <&gic>;
+	interrupts = <0 92 4>;
+	reset-gpios = <&gpio 80 1>;
+	reg = <0x0 0x80000000 0x0 0x10000>;
 };
-- 
2.7.4

